devbox.go

```diff
@@ -340,6 +347,17 @@ func (d *Devbox) PrintShellEnv() error {
 	return nil
 }
 
+func (d *Devbox) Info(pkg string) error {
+	hasReadme, err := pkgcfg.PrintReadme(pkg, d.configDir, d.writer)
+	if err != nil {
+		return err
+	}
+	if !hasReadme {
+		_, err = fmt.Fprintf(d.writer, "No info available for %s\n", pkg)
+	}
+	return errors.WithStack(err)
```

Comment:
hmm the `devbox info <pkg>` command is a bit confusing. I'd expect the `info` command to detail information about any nix package, and additionally any devbox-package (what are we calling it?).

But it does only the latter, so it would feel confusing or incomplete to the user.

Is there a plan to add info about nix packages as well?

Reply:
@Lagoja @loreto what do you think?